draft-ietf-rats-corim.md

```diff
@@ -292,21 +292,20 @@ The CDDL definitions in this document follows the naming conventions illustrated
 
 A CoRIM is a collection of tags and related metadata in a concise CBOR {{-cbor}} encoding.
 A CoRIM can be digitally signed with a COSE {{-cose}} signature.
-A tag identifies and describes properties of modules or components of a system.
+A tag is a structured, machine-readable metadata format used to uniquely identify, describe, and manage modules or components of a system.
 
 Tags can be of different types:
 
 * Concise Module ID (CoMID) tags ({{sec-comid}}) contain metadata and claims about the hardware and firmware modules.
 
-* Concise Software ID (CoSWID) tags ({{-coswid}}) describe software components.
+* Concise Software ID (CoSWID) tags ({{-coswid}}) uniquely identify, describe, and manage software components.
 
 * Concise Tag List (CoTL) tags ({{sec-cotl}}) contain the list of CoMID and CoSWID tags that the Verifier should consider as "active" at a certain point in time.
 
-The set of tags is extensible so that future specifications can add new kinds of information.
+The collection of tags is extensible so that future specifications can add new types of tags.
```

Comment:
I don't think "collection" is the right word in this context.  I suggest a more concise rephrasing that works around that.

```suggestion
CoRIM allows for new types of tags to be added in future specifications.
```

Reply:
This happens very early in the document.  At this point, we just need to state the high-level features and avoid any information overload.  The reader has not yet an idea of what _"extensible tag type"_ means.